dotcom-rendering/src/model/enhance-H2s.ts

```diff
@@ -1,5 +1,13 @@
 import { JSDOM } from 'jsdom';
 
+const hasInteractiveContentsElement = (elements: CAPIElement[]): boolean => {
+	return !!elements.find(
+		(element) =>
+			element._type ===
+			'model.dotcomrendering.pageElements.InteractiveContentsBlockElement',
+	);
```

Comment:
Might be more semantic to use `some`?

```suggestion
	return elements.some(
		(element) =>
			element._type ===
			'model.dotcomrendering.pageElements.InteractiveContentsBlockElement',
	);
```

Reply:
Nice!